explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -186,6 +192,19 @@ explorerHandlers _sendActions =
 -- API Functions
 ----------------------------------------------------------------
 
+getTotalAda :: ExplorerMode ctx m => m CCoin
+getTotalAda = do
+    utxoSum <- EX.getUtxoSum
+    -- getUtxoSum returns Integer, but for external consumers we want to provide Coin.
+    -- However, we don't want to return meaningless values to them, so we add truncation.
+    pure $ mkCCoin $ truncateIntegerToCoin utxoSum
```

Comment:
By @Pastafarianist: `totalAda` endpoint returns lovelaces?

Reply:
good point. Do we return total ada or total lovelaces ?